pages/[league]/player/[id].tsx

```diff
@@ -0,0 +1,337 @@
+
+  const {
+    ratings: skaterStats,
+    isLoading: isLoadingSkaterStats,
+    isError: isErrorSkaterStats,
+  } = useSkaterStatsId(id, league, filterSeasonType);
+
+  // player info
+  const {
+    ratings: goalieInfo,
+    isLoading: isLoadingGoalieInfo,
+    isError: isErrorGoalieInfo,
+  } = useGoalieInfo(id, league)
+
+  const {
+    ratings: skaterInfo,
+    isLoading: isLoadingSkaterInfo,
+    isError: isErrorSkaterInfo,
+  } = useSkaterInfo(id, league)
+
+  // wait for all loads to complete
+  useEffect(() => {
+    setIsLoading(isLoadingGoalieRating || isLoadingPlayerRating || isLoadingGoalieStats
+      || isLoadingSkaterStats || isLoadingGoalieInfo || isLoadingSkaterInfo);
+    if (!isLoading) {
+      if (isErrorGoalieRating || isErrorPlayerRating || isErrorGoalieStats
+        || isErrorSkaterStats || isErrorSkaterInfo || isErrorGoalieInfo) {
+        setPlayerError(true);
+      } else {
+        if (skaterStats !== undefined && skaterInfo !== undefined) {
+          if (skaterStats.length > 0) {
+            setIsSkater(true);
+            setPlayerName(skaterStats[0].name);
+            setPlayerPosition(skaterStats[0].position);
+            setPlayerTeam(skaterStats[0].team.toString());
+            setPlayerHeight(skaterInfo[0].height.toString());
+            setPlayerWeight(skaterInfo[0].weight.toString());
+          } else {
+            setIsSkater(false);
+            setPlayerName(goalieStats[0].name);
+            setPlayerPosition('G');
+            setPlayerTeam(goalieStats[0].team.toString());
+            setPlayerHeight(goalieInfo[0].height.toString());
+            setPlayerWeight(goalieInfo[0].weight.toString());
+          }
+        }
+      }
+    }
+  });
+
+  const onSeasonTypeSelect = async (seasonType: SeasonType) => {
+    setFilterSeasonType(seasonType);
+  };
+
+  return (
+    <React.Fragment>
+      <NextSeo
+        title="Player"
+        openGraph={{
+          title: 'Player',
+        }}
+      />
+      <Header league={league} activePage="players" isSticky={false} />
+      <Container>
+        {isLoading && !playerError && (
+          <CenteredContent>
+            <PulseLoader size={15} />
+          </CenteredContent>
+        )}
+        <Main>
+          {playerError && (
+            <ErrorBlock>
+              Failed to load player info. Please reload the page to try again.
+            </ErrorBlock>
+          )}
+          <Table>
+            <TableRow>
+              <TableData>
+                <Button onClick={() => router.back()} inverse>
+                  <ButtonContent>
+                    Go Back
+                  </ButtonContent>
+                </Button>
+              </TableData>
+              <TableData>
+                <SelectorWrapper>
+                  <SeasonTypeSelector onChange={onSeasonTypeSelect} />
+                </SelectorWrapper>
+              </TableData>
+            </TableRow>
+          </Table>
```

Comment:
Is there a good reason we are using a table here? What are you looking to do with this? I'm 100% sure there is a better way to do this because we shouldn't use tables for positioning content.